code/game/objects/items/weapons/weaponry.dm

```diff
@@ -221,7 +221,7 @@
 
 /obj/item/weapon/cane
 	name = "cane"
-	desc = "A cane used by a true gentlemen. Or a clown."
+	desc = "A cane used by a true gentleman. Or a clown."

```

Comment:
while I was in the file I fixed this grammatical error, I am literally hitler this is gamebreaking